src/d3a/models/power_flow/__init__.py

```diff
@@ -55,7 +55,7 @@ def _d3a_area_to_electrical_grid(self, area):
         elif isinstance(area.strategy, PVStrategy):
             area.pv_device = \
                 self.add_generation_device(area,
-                                           convert_unit_to_kilo(area.strategy.max_panel_power_W))
+                                           convert_unit_to_kilo(area.strategy.capacity_kW))
```

Comment:
If capacity is already in kWh, this shouldn't be necessary, right?

Reply:
Wow, good catch! Thanks!

Reply:
Sure! It would be great to have some tests that would fail with the previous bug, if it's not too convoluted with the current structure.

Reply:
This is only the power flow calculation that is not used operationally ATM but is tested in two integrationtests. I removed the W->kW conversion but don't plan to add another test.